src/redux/categories/categories.js

```diff
@@ -0,0 +1,24 @@
+// Actions
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/214248783-231ad861-a914-4c52-bf8f-dac3b3533226.png)

**[Optional]**

- [ ] Your branch name is **reducer** which is fair enough :+1: but it would be if you make it more descriptive such as **feature/add-reducers-actions** :+1:

**Important Note: If you rename your branch then this pull request will be closed and you will need to open a new pull request from the renamed branch :thinking: So if you do it then it would be nice if you add the link of your closed pull request to the summary of new pull request :+1: If you don't wanna do it then count this requirement in your next milestone but keep in mind that your pull request shows your professionalism and passion with your work so please be careful :blush:**